src/gtfs/repository/ScheduleBuilder.ts

```diff
@@ -94,6 +94,21 @@ export class ScheduleBuilder {
     );
   }
 
+  private getActivities(activity: string) {
+    let activitiesPairs = activity.match(/.{1,2}/g);
```

Comment:
if I understand it correctly, you are trying to check if it's 1-2 chars long - can't you just do `"".length()` ?

Reply:
I divide "TFK" or "TFT " to array of elements where each element has 2 characters.
I forgot to copy unit test from journey planner.

In a minute I'll add one but you can also take a look at old closed PR
https://github.com/assertis/journey-planner/pull/40/files#diff-96361a5cb388352dbd22a779be19bba5R166

And this
https://github.com/assertis/journey-planner/pull/40/files#diff-d7a46dc9c97b5c58a08fb9209fb1bb60R16

Reply:
Unit test added https://github.com/assertis/dtd2mysql/pull/9/files#diff-534c3f685a4c9ac4758665d20e337a94R1